desktop/src/main/java/bisq/desktop/main/market/trades/TradesChartsView.java

```diff
@@ -327,7 +329,8 @@ private CurrencyListItem specialShowAllItem() {
     // Chart
     ///////////////////////////////////////////////////////////////////////////////////////////
 
-    private void createCharts() {
+    @SuppressWarnings("unchecked")
+	private void createCharts() {
```

Comment:
formatting seems off

```suggestion
    private void createCharts() {
```